client/packages/system/src/Patient/ListView/ListView.tsx

```diff
@@ -198,8 +218,4 @@ const PatientListComponent = () => {
   );
 };
 
-export const PatientListView = () => (
-  <TableProvider createStore={createTableStore}>
-    <PatientListComponent />
-  </TableProvider>
-);
+export const PatientListView = () => <PatientListComponent />;
```

Comment:
Don't need this wrapper, just rename PatientListComponent to PatientListView and export it directly